regions_www/m/server.R

```diff
@@ -252,11 +273,18 @@ shinyServer(function(input, output, session) {
 
     # Identify region from URL or use a default
     if (is.na(region$current)) {
-      query <- parseQueryString(session$clientData$url_search)
-      region$current <- if (isTRUE(query[['r']] %in% regions$region_name)) {
-        query[['r']]
-      } else {
-        "isle-of-wight"
+
+      if(!is.null(region$state_region) && !is.na(region$state_region)){
```

Comment:
Why we need to do both `na` and `null` checks?  I get confused which to use in `R`...

Reply:
`null` when it's not initialized - ran before the initialization of the `region` variable, and `na` when there is no saved state.